src/base/static/components/organisms/map-legend-panel.js

```diff
@@ -6,22 +6,22 @@ import { Image } from "../atoms/imagery";
 import { Paragraph, Header4 } from "../atoms/typography";
 import MapLegendGroup from "../molecules/map-legend-group";
 
-const MapLegendPanelContainer = styled("div")({
+const MapLegendPanelContainer = styled("div")(props => ({
   padding: 10,
+  backgroundColor: props.themed ? props.theme.brand.secondary : "#fff",
   margin: 0,
-});
+}));
 
 const MapLegendPanel = props => {
   return (
-    <MapLegendPanelContainer>
+    <MapLegendPanelContainer themed={props.themed}>
```

Comment:
The idea here is that we can render this component in "themed" mode (where it will adopt coloring from the `theme.brand` section of the config), or in regular mode, where it won't.